db/data.py

```diff
@@ -483,8 +489,10 @@ def get_unprocessed_highlevel_documents():
             LEFT JOIN highlevel AS hl
                    ON ll.id = hl.id
                 WHERE hl.mbid IS NULL
+                  AND ll.id >= :min_row_id
+             ORDER BY ll.id
```

Comment:
does this give more or less the same results as without the `ORDER BY`? part of what we're trying to do here is reduce the amount of work that the query has to do. Instinctively I think it doesn't matter _what_ ids we get as long as they're above min_row_id.

@mayhem had a thought that performing operations such as join and sort without the lowlevel_json table would be a bit faster. We saw that with queries that return a lot of data (such as this one, joining against lowlevel_json) often have to use the disk to perform a sort/order because there's not enough free memory. I worry that adding a sort but keeping lowlevel_json will cause this to happen again.

Can we perform some additional query plans (using explain analyze to get real results, instead of just expected plans),

* select only lowlevel and have an order by
* select lowlevel + lowlevel_json with order by
* the two above without order by

If the query plan without lowlevel_json is much better, we'll also have to check that selecting just 200-ish items from lowlevel_json in a second query will also not take much time. Again, I believe that this should be fine with our index - we'll be doing no complex sorts or filtering.

The query plan looks ok - I see that the WHERE on ll.id happens as the inner-most operation, so I think that this means that it will restrict the number of rows that we have to deal with when we do the left join and where hl.mbid is null. This is good. To confirm, can we do an `explain analyze` with and without the ll.id limit to verify that the execution speed changes?